lua/orgmode/capture/init.lua

```diff
@@ -343,16 +304,43 @@ function Capture:_refile_to(opts)
     return false
   end
 
-  apply_properties(opts)
+  local has_headline = opts.headline and opts.headline ~= ''
+  local destination_file = Files.get(opts.file)
+  local target_level
+  local target_line
```

Comment:
Define these as defaults, and then we can remove the `else` block.
```suggestion
  local target_level = 0
  local target_line = -1
```

Reply:
Done.